### PR TITLE
Feature/update schema

### PR DESCRIPTION
**Pull request by brandonzhu09**
## Description
A revised version of the input tool with an updated schema to reflect its use in the Django application.

### CODE REVIEW

**Review by asewnath — COMMENTED** [review]
Looks good to me, I'll wait for Tariq to take a look at this before we decide to merge it

**Review by ghost — APPROVED** [review]

**Pull request merged**
